packages/wallet/src/wallet.test.ts

```diff
@@ -61,6 +62,75 @@ describe('Pactus Wallet Tests', () => {
             expect(restoredWallet.getMnemonicWordCount()).toBe(12);
         });
 
+        it('should restore a wallet from external BIP39 mnemonic correctly', async () => {
+            // Generate a random BIP39 mnemonic with 12 words (128 bits of entropy)
+            const externalBip39Mnemonic = bip39.generateMnemonic(128);
```

Comment:
use this mnemonic as test:
`"abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon cactus"`

It should generate following addresses:

```
pc1rcx9x55nfme5juwdgxd2ksjdcmhvmvkrygmxpa3
pc1r7aynw9urvh66ktr3fte2gskjjnxzruflkgde94
```